tool/src/main/java/org/wildfly/security/tool/VaultCommand.java

```diff
@@ -85,6 +85,7 @@ public class VaultCommand extends Command {
     public static final String VAULT_COMMAND = "vault";
 
     public static final String FAIL_IF_EXIST_PARAM = "fail-if-exist";
+    public static final String MASK_PREFIX = MASK_PREFIX;
```

Comment:
@VAIBHAVCES Just a minor, this will not build as you are assigning a nonexistent variable MASK_PREFIX to the MASK_PREFIX as a self reference.
This needs to be `public static final String MASK_PREFIX = "MASK-";`

Reply:
you can build the project with running `mvn clean install` and see if your changes can be built successfully